Sources/PackagePlugin/PluginContextDeserializer.swift

```diff
@@ -278,190 +248,6 @@ fileprivate struct PluginInputDeserializer {
     }
 }
 
-/// The input structure received as JSON from SwiftPM, consisting of an array
```

Comment:
This is moved out into PluginMessages, and is now shared between the plugin side and the host side.